### PR TITLE
UBER-135/TSK-1430: allow changing image in PDFViewer through arrow-keys (keyboard)

### PR DESCRIPTION
**Pull request by ThetaDR**
# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues
https://front.hc.engineering/workbench/platform/tracker/TSK-1430

A list of closed updated issues

### CODE REVIEW

File: packages/ui/src/popups.ts [hunk 1]
```diff
@@ -32,6 +32,16 @@ export interface PopupResult {
 
 export const popupstore = writable<CompAndProps[]>([])
 
+export function updatePopup (id: string, props: any): void {
```
Comment (haiodo) [inline packages/ui/src/popups.ts]:
```suggestion
export function updatePopup (id: string, props: CompAndProps): void {
```
?

File: plugins/attachment-resources/src/components/AttachmentPreview.svelte [hunk 2]
```diff
@@ -34,11 +38,13 @@
...
     on:click={() => {
       closeTooltip()
-      showPopup(
+      if (listProvider !== undefined) listProvider.updateFocus(value)
+      const popupInfo = showPopup(
         PDFViewer,
         { file: value.file, name: value.name, contentType: value.type },
         value.type.startsWith('image/') ? 'centered' : 'float'
       )
+      dispatch('popupOpened', popupInfo.id)
```
Comment (haiodo) [inline plugins/attachment-resources/src/components/AttachmentPreview.svelte]:
```suggestion
      dispatch('open', popupInfo.id)
```

File: plugins/attachment-resources/src/components/AttachmentStyledBox.svelte [hunk 3]
```diff
@@ -113,6 +126,7 @@
         }
       )
     }
+    console.log(attachments)
```
Comment (haiodo) [inline plugins/attachment-resources/src/components/AttachmentStyledBox.svelte]:
```suggestion
```

File: packages/ui/src/popups.ts (hunk 1 above)
Comment (ThetaDR) [reply packages/ui/src/popups.ts]: Then we would have to get existing CompAndProps for the popup and pass all of them as an argument. This way we can just use popupId and new props for popup's component

Comment (BykhovDenis) [reply packages/ui/src/popups.ts]: You can use Partial

Comment (ThetaDR) [reply packages/ui/src/popups.ts]: Made it partial

**Review by haiodo — APPROVED** [review]

**Pull request merged**